protocols/secio/benches/bench.rs

```diff
@@ -18,106 +18,141 @@
 // FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 // DEALINGS IN THE SOFTWARE.
 
-use criterion::{Bencher, Criterion, criterion_main, criterion_group};
+use criterion::{criterion_group, criterion_main, Bencher, Criterion};
 use futures::prelude::*;
-use libp2p_core::Transport;
-use tokio::{
-    io,
-    runtime::current_thread::Runtime
-};
+use libp2p_core::{Transport, transport::upgrade::Version};
 
-fn secio_and_send_data(bench: &mut Bencher, data: &[u8]) {
-    let key = libp2p_secio::SecioKeyPair::ed25519_generated().unwrap();
-    let transport =
-        libp2p_tcp::TcpConfig::new().with_upgrade(libp2p_secio::SecioConfig::new(key));
+fn secio_and_send_data(bench: &mut Bencher<'_>, data: &[u8]) {
+    // let key = libp2p_core::identity::Keypair::generate_ed25519();
```

Comment:
Why check-in this commented-out code?

Reply:
The original code doesn’t compile, and I wasn’t able to fix it due to how `Upgrade` works.